docs/img-src/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Do you know a better alternative for mermaid? I would love to have something with a source code, because it makes the whole thing extensible to other developers. (in contrast to a PS file or something)